### PR TITLE
[DOC] missing get_test_params in transformer extension template

### PR DESCRIPTION
**Pull request by fkiraly**
This PR adds the missing `get_test_params` function in the transformer extension template.

Also cleans up the preambles of the other templates so it's clear what needs to be implemented.

### CODE REVIEW

**Comment by TonyBagnall** [comment]
this still branch still has the old classifier tags in it. these have changed to "X_inner_mtype": "numpy3D",

**Comment by fkiraly** [comment]
well, that should be resolved with an update from `main`, doing this now

**Comment by fkiraly** [comment]
done, is it resolved by update from main? If not, then you (or I) need to update the tag names in a separate PR.

**Comment by fkiraly** [comment]
@TonyBagnall, your issue should be fixed, kindly have a look.

File: extension_templates/transformer.py [hunk 1]
```diff
@@ -241,3 +235,42 @@ def _inverse_transform(self, X, y=None):
...
+        """Get fitted parameters.
+
+        Returns
+        -------
+        fitted_params : dict
+        """
+        # implement here
+
+    # todo: return default parameters, so that a test instance can be created
+    @classmethod
```
Comment (chrisholder) [inline extension_templates/transformer.py:251]: Make this static rather than class you don't use the cls instance in any of the examlpes i've seen

**Review by chrisholder — CHANGES_REQUESTED** [review]

**Review by chrisholder — APPROVED** [review]

**Pull request merged**